src/text-editor.coffee

```diff
@@ -892,7 +892,7 @@ class TextEditor extends Model
   # Determine whether the user should be prompted to save before closing
   # this editor.
   shouldPromptToSave: ({windowCloseRequested, projectHasPaths}={}) ->
-    if windowCloseRequested and projectHasPaths
+    if windowCloseRequested and projectHasPaths and atom.stateStore.isConnected
```

Comment:
OK the only other reference I could find is on line 821, which is actually a global reference:

```
for textEditor in atom.workspace.getTextEditors() when textEditor isnt this
```

So I'm ignoring this. 😈